### PR TITLE
feat(lpa request new code): added new feature to allow LPAs to Request a New Code

### PR DESCRIPTION
**Pull request by suityou01**
## Ticket Number

AS-5883

https://pins-ds.atlassian.net/browse/AS-5883

## Description of change

Added route, view and controller (and associated unit tests) for the LPA Request New Code story.

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/views/manage-appeals/request-new-code.njk [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      <form action="" method="post" novalidate>
+         {% call govukFieldset({
+          legend: {
+            text: "Enter your email address",
+            classes: "govuk-fieldset__legend--l",
+            isPageHeading: true
+          }
+        }) %}
+
+        {{ govukInput({
```
Comment (bryn-pins) [inline packages/forms-web-app/src/views/manage-appeals/request-new-code.njk:35]: Should we be having an email on this field.
Think we should already have this available, unless this is the entry point.

In my head we were still waiting for a story for entering email, can see you left a message in teams about this so will double check my assumptions

File: packages/forms-web-app/src/controllers/common/request-new-code.js [hunk 2]
```diff
@@ -15,7 +23,43 @@ const postRequestNewCode = (enterCodeView) => {
...
+
+		let user;
+
+		try {
+			user = await getUserByEmail(emailAddress);
+		} catch (e) {
+			logger.info(e.message);
+		}
+
+		if (!user) {
```
[the remark this answers is not in the document]
Comment (suityou01) [reply packages/forms-web-app/src/controllers/common/request-new-code.js:48]: I need to check my understanding of which logger transports are configured. If there is a console transport then yes, this makes sense.

File: packages/forms-web-app/src/views/manage-appeals/request-new-code.njk (hunk 1 above)
Comment (suityou01) [reply packages/forms-web-app/src/views/manage-appeals/request-new-code.njk:35]: This was requested by Paul G and Chris Hy

**Review by bryn-pins — APPROVED** [review]

**Review by chrishawldev — APPROVED** [review]

**Review by chrishawldev — APPROVED** [review]

**Comment by suityou01** [comment]
Rebase and merge

**Pull request merged**